PRTest/PRTest/ContentView.swift

```diff
@@ -9,8 +9,22 @@ import SwiftUI
 
 struct ContentView: View {
     
+    @State private var randomNumber: Int = 0
+    
     var body: some View {
-        Text("한톨민톨")
+        VStack {
+            Text("\(randomNumber)")
+            
+            Button("랜덤 숫자 돌리기") {
+                fetchRandomNumber()
+            }
+        }
+        .padding()
+    }
+    
+    /// 랜덤 숫자를 반환합니다.
+    private func fetchRandomNumber() {
+        randomNumber = Int.random(in: 1...100)
     }
```

Comment:
```suggestion
    private func fetchRandomNumber(from min: Int, to max: Int) {
        randomNumber = Int.random(in: min...max)
    }
```

범위 값을 메소드의 매개변수로 받는건 어떻게 생각하시나요?